Jenkinsfile

```diff
@@ -104,7 +114,8 @@ if( buildParams.BUILD_OS != "all" ){
 
 // Using .each{} hits "a CPS-transformed closure is not yet supported (JENKINS-26481)"
 for( label in build_nodes) {
-  stage_unit_test(label)
-  stage_rpmbuild(label)
-  stage_integration(label)
+//stage_unit_test(label)
+//stage_rpmbuild(label)
+//stage_integration(label)
+  stage_clean_rpms(label)
 }
```

Comment:
Do you mean, why did I move the file? I just thought it made more sense to reside in the ci/repo_cleanups directory, since that is what it is logically connected to. The jenkins configuration was updated to reflect this new location.